<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="12" column="1">
      <widget class="QPushButton" name="pushButton_rotate">
       <property name="text">
        <string>Повернуть</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_translation">
       <property name="text">
        <string>Перенос</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>120</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_translation">
       <property name="text">
        <string>Переместить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_load">
       <property name="placeholderText">
        <string>input.txt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_load">
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_load">
       <property name="text">
        <string>Загрузка модели</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="lineEdit_dx">
       <property name="placeholderText">
        <string>dx</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLineEdit" name="lineEdit_dz">
       <property name="placeholderText">
        <string>dz</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_rotation">
       <property name="text">
        <string>Поворот (град.)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_save">
       <property name="placeholderText">
        <string>output.txt</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="lineEdit_dy">
       <property name="placeholderText">
        <string>dy</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_save">
       <property name="text">
        <string>Сохранение модели</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>OX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OZ</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="14" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="0">
      <widget class="QLineEdit" name="lineEdit_angle">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>alpha</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_scaling">
       <property name="text">
        <string>Масштабирование</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLineEdit" name="lineEdit_scale">
       <property name="placeholderText">
        <string>k</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Масштабировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
